src/csvToJson.js

```diff
@@ -24,7 +25,7 @@ const fileWriter = (i, fileName, start, stop) => {
   // @TODO change that. it will work only for one case.
   // we can also create a method for path.join, so it wouldn't complicate our code
   // really bad line
-  const jsonPath = `/projects/USFA/${folderName}/${fileName}${i}.json`;
+  const jsonPath = `/projects/FAO/${folderName}/${fileName}${i}.json`;
```

Comment:
you should update this line.
As you can see - it was working with USFA before. When you change foldername - you destroying previous working script.